src/docs/admin/references/NotificationServiceReference.adoc

```diff
@@ -96,9 +100,16 @@ if not provided the extension of the file will determine the appropriate engine
 notification.script.engine=bash
 ----
 
+[[multiple-execution]]
 Finally, multiple sequential script execution can be enabled.
-Users must have an active subscription for an event and the Third party notification method activated.
-It is set to false by default.
+When activated, each notified user will trigger a script execution (instead of one execution per notification)
```

Comment:
something is not clear. This will trigger a single script execution for all notifications possibly occurring for a user ?
So if a user received a notification for one job, then 2 minutes later another notification for another job, it will trigger only a single script execution ???